README.md

```diff
@@ -324,3 +324,47 @@ Please refer to mocha documentation to know how to use them.
 
 If you want to use that with storybook, you'll need to add them to your mocha config and storybook config files.
 You can find the complete configuration by looking at the [samples directory](https://github.com/mthuret/storybook-addon-specifications/tree/master/.storybook)
+
+#### Loading External Test Files
```

Comment:
Maybe we could use this title as new big category and add it to the TOC? 

Also, wdyt about something like : 'Adding tests from external files to your stories.' ?